src/contract/default.ts

```diff
@@ -87,10 +95,25 @@ function buildEstimate(contract: Contract, functionAbi: FunctionAbi): ContractFu
   };
 }
 
-const detectCairoVersion = (abi: Abi) => {
+export function getCalldata(
+  args: RawArgsArray | [Calldata] | Calldata,
+  callback: Function
+): Calldata {
```

Comment:
```suggestion
export function getCalldata(args: ArgsOrCalldata, callback: Function): Calldata {
```


When does the need for the `[Calldata]` segment arise? Would it make sense to not support it?